### PR TITLE
update instructions as per the latest public repos

### PR DESCRIPTION
**Pull request by mycpuorg**
* update tags in docker readme
* update conda binary names and pip repo names

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +57,12 @@ _macOS_
 1. Create an environment and install torchserve and torch-model-archiver
     ```bash
     conda create --name torchserve torchserve torch-model-archiver -c pytorch
+    conda install -c pytorch -c powerai pytorch torchtext torchvision
+    ```
+    For GPU
```
Comment (aaronmarkham) [inline README.md:62]: Is mac/GPU tested?

File: README.md [hunk 2]
```diff
@@ -51,6 +57,12 @@ _macOS_
 1. Create an environment and install torchserve and torch-model-archiver
     ```bash
     conda create --name torchserve torchserve torch-model-archiver -c pytorch
+    conda install -c pytorch -c powerai pytorch torchtext torchvision
```
Comment (aaronmarkham) [inline README.md]: Needs one liner.

File: README.md (hunk 1 above)
Comment (mycpuorg) [reply README.md]: good catch

File: README.md [hunk 3]
```diff
@@ -47,10 +53,10 @@ _macOS_
     brew tap AdoptOpenJDK/openjdk
     brew cask install adoptopenjdk11
     ```
-1. Install Conda (https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html)
+1. Install Conda (https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html)
 1. Create an environment and install torchserve and torch-model-archiver
     ```bash
-    conda create --name torchserve torchserve torch-model-archiver -c pytorch
+    conda create --name torchserve torchserve torch-model-archiver pytorch torchtext torchvision -c pytorch -c powerai
```
Comment (zhangguanheng66) [inline README.md:59]: Is `sentencepiece` missing?

**Pull request merged**